### PR TITLE
chore: Use @excalidraw/eslint-config

### PR DESCRIPTION
**Pull request by lipis**
Experimental

- https://www.npmjs.com/package/@excalidraw/eslint-config
- https://github.com/excalidraw/eslint-config

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,40 +1,6 @@
...
-    "prefer-arrow-callback": "warn",
-    "prefer-const": [
-      "warn",
-      {
-        "destructuring": "all"
-      }
-    ],
-    "prefer-template": "warn",
-    "prettier/prettier": "warn"
+    "import/no-anonymous-default-export": "off"
```
Comment (dwelle) [inline .eslintrc.json:4]: why not put this in the config too?

Comment (lipis) [reply .eslintrc.json:4]: That's a good question.. I didn't include the `react-app` related plugins and deps in the core.. as they only needed in this repo..

And that was the only rule from this list.. If you ask me, we should remove it and comply with that rule eventuall.

Comment (dwelle) [reply .eslintrc.json:4]: Is it React related tho? Shouldn't it apply to exports in general?

Comment (lipis) [reply .eslintrc.json:4]: ¯\_(ツ)_/¯

**Comment by lipis** [comment]
https://github.com/excalidraw/eslint-config/network/dependents

**Review by dwelle — APPROVED** [review]

**Pull request merged**